volontulo/templates/offers/offer_apply.html

```diff
@@ -14,7 +14,7 @@
             </div>
             <label class="col-xs-1 control-label" for="{{ form.email.id_for_label }}">Email:</label>
             <div class="col-xs-4">
-               <input type="text" value="{{ request.POST.email|default:volunteer_user.user.email }}" name="email" class="form-control" id="{{form.email.id_for_label }}"{% if user.is_authenticated %}disabled="disabled"{% endif %} />
```

Comment:
We can also write all code as small as possible like below:
`
<!doctype html>
<html lang=en>
<meta charset=utf-8>
<title>blah</title>
<p>I'm the content
`
It's valid but it's far from clear code.